cypress.json

```diff
@@ -0,0 +1,7 @@
+{
+  "integrationFolder": "tests/e2e/integration",
+  "supportFile": "tests/e2e/support",
+  "fixturesFolder": "tests/e2e/fixtures",
+  "pluginsFile": "tests/e2e/plugins",
+  "reporter": "mochawesome"
```

Comment:
I'd suggest defining here paths for screenshots and videos on custom one, like `artifacts/`. It's more readable than `/cypress` IMHO